workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/ScreenComposableFactoryFinder.kt

```diff
@@ -0,0 +1,69 @@
+package com.squareup.workflow1.ui.compose
+
+import com.squareup.workflow1.ui.EnvironmentScreen
+import com.squareup.workflow1.ui.NamedScreen
+import com.squareup.workflow1.ui.Screen
+import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.ViewEnvironmentKey
+import com.squareup.workflow1.ui.ViewRegistry
+import com.squareup.workflow1.ui.ViewRegistry.Key
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.getFactoryFor
+
+@WorkflowUiExperimentalApi
+public interface ScreenComposableFactoryFinder {
+  public fun <ScreenT : Screen> getComposableFactoryForRendering(
+    environment: ViewEnvironment,
+    rendering: ScreenT
+  ): ScreenComposableFactory<ScreenT>? {
+    val factoryOrNull: ScreenComposableFactory<ScreenT>? =
+      environment[ViewRegistry].getFactoryFor(rendering)
+
+    @Suppress("UNCHECKED_CAST")
+    return factoryOrNull
+      ?: (rendering as? ComposeScreen)?.let {
+        ScreenComposableFactory<ComposeScreen> { rendering, environment ->
+          // Do we need to key on either, both params?
```

Comment:
Do we need to answer this before merging?

Reply:
Answer appears to be no, deleting.